Clarifying consensus about role selection (#2382)
kubernetes/sig-release: release-team/release-team-selection.md

### Release Team Lead

Consensus must be reached about the next Release Team Lead (RT Lead) by:

- [SIG Release Leads](https://git.k8s.io/community/sig-release/#leadership)
- The current Release Team Lead
- The current Emeritus Advisor

The incoming Release Team Lead _MUST_ have participated on the Release Team for two or more release cycles, acting in a lead (non-shadow) or Release Team Lead shadow capacity for at least one of those cycles.

Release Team Leads should be staffed, with this order of fall-through in mind:

- the current pool of Release Team Lead shadows
- the current pool of Release Team members
- former Release Team members

Bear in mind that these are suggestions based on precedent and a Release Team Lead may be nominated by any Release Team member, past or present.

#### Selection Process

> At least 2 weeks before the new release cycle begins, the EA is responsible for

Reviewer: Two weeks before the beginning of the next release cycle is too tight to give adequate time for subteam shadow applications and selection, and isn't when this has been happening anyway. Thoughts on this change? It's more in line with what actually happens. 

```suggestion
At least 2 weeks before the end of the current release cycle, the EA is responsible for
```

Reviewer 2: Looks good. Thanks for the review, Kat!

### Release Team Lead

> Consensus must be reached about the next release team lead by:

Reviewer 2: ```suggestion
Consensus must be reached about the next Release Team Lead (RT Lead) by:
```

> - [SIG Release Leads](https://github.com/orgs/kubernetes/teams/sig-release-leads)

Reviewer 2: ```suggestion
- [SIG Release Leads](https://git.k8s.io/community/sig-release/#leadership)
```

(GitHub team will not be visible to non-org members)
